go/config/config.go

```diff
@@ -608,9 +630,67 @@ func (config *ClusterClientConfiguration) GetSubscription() *ClusterSubscription
 	return nil
 }
 
+// TlsConfiguration represents TLS-specific configuration settings.
+type TlsConfiguration struct {
```

Comment:
Is insecure TLS not supported in Go?

Reply:
It was missed somehow. I added the feature in this PR